Userland/Applications/MailSettings/MailSettingsWindow.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021, The SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGUI/TextEditor.h>
+#include <LibGUI/Window.h>
+
+class MailSettingsWindow final : public GUI::Window {
+    C_OBJECT(MailSettingsWindow)
+public:
+    virtual ~MailSettingsWindow() override;
+
+private:
+    MailSettingsWindow();
+
+    void reset_default_values();
+    void write_values();
+
+    String m_server { "" };
+    int m_port { 0 };
+    bool m_tls { false };
+    String m_email { "" };
```

Comment:
```suggestion
    String m_server;
    int m_port { 0 };
    bool m_tls { false };
    String m_email;
```

Non-primitive types like `String` don't need explicit initialization.